source/kraken/configuration.cpp

```diff
@@ -59,7 +59,9 @@ po::options_description get_options_description(const boost::optional<std::strin
         ("GENERAL.is_realtime_enabled", po::value<bool>()->default_value(false),
                                         "enable loading of realtime data")
         ("GENERAL.is_realtime_add_enabled", po::value<bool>()->default_value(false),
-                                        "enable loading of realtime data that add stop_times or trips")
+                                        "enable loading of realtime data that add stop_times")
+        ("GENERAL.enable_realtime_add_trip", po::value<bool>()->default_value(false),
```

Comment:
I'd rather have the same naming convention (it will appear in conf also), so is_realtime_`trip_`add_enabled (or change the other ones, but it will require changing the conf also).
Then it will other names also in this PR (to be checked exhaustively).

Reply:
We have decided to call the parameter `enable_realtime_add_trip`. I follow the rules in the JIRA ticket :smile_cat:

Reply:
As long as it doesn't carry crap around, I agree with PE that consistency is usually best keeping ! 
I would be tempted to name the option `is_realtime_add_trip_enabled`.

I just wonder if `is_realtime_add_enabled` still makes sense though, and would rather be renamed `is_realtime_add_stop_enabled` ?

Reply:
Okay, so we need to update all deployment conf with that.

Reply:
To me, `is_realtime_add_enabled` can stay the same and it would save some work on conf's PR.
OK anyway.

Reply:
after debat :
- is_realtime_enabled
- is_realtime_add_enabled
- is_realtime_add_trip_enabled